interlok-core/src/main/java/com/adaptris/sftp/SftpClient.java

```diff
@@ -99,15 +117,30 @@ private SftpClient(File knownHostsFile, ConfigBuilder configBuilder) throws Sftp
    *
    * @param host the remote ssh host.
    */
+  @Generated
   public SftpClient(String host) throws SftpException {
-    this(host, DEFAULT_SSH_PORT, DEFAULT_TIMEOUT, null, null);
+    this(host, DEFAULT_SSH_PORT);
+  }
+
+  /**
+   * Constructor assuming the default SSH port.
+   *
+   * @param host the remote ssh host.
+   * @param port the ssh port.
+   */
+  @Generated
+  public SftpClient(String host, int port) throws SftpException {
+    this(host, port, DEFAULT_TIMEOUT);
   }
 
   /**
    * Constructor assuming the default SSH port.
    *
    * @param addr the remote ssh host.
+   * @deprecated since 4.5.0
```

Comment:
We want "since 4.4.0" I guess if we make it to the next release?

Reply:
TBH It doesn't matter, nothing actually uses it (!) but I will change it anyway, at the time I thought 4.4 was the next release.